scipy/special/amos_wrappers.c

```diff
@@ -5,6 +5,14 @@
 #define CMPLX(x, y) ((double complex)((double)(x) + I * (double)(y)))
 #endif /* CMPLX */
```

Comment:
Sorry. I'm still confused. Why are we touching the amos_wrappers files at all still? I think this PR should leave both `amos_wrappers.c` and `amos_wrappers.h` untouched.

Reply:
Right, there are absolutely no changes in amos_wrappers now.

Reply:
Ah. Sorry, there was something weird with GitHub that made it seem like they were still there.